releasenotes/notes/matplotlib_deprecate_fix_11144-914a20fc305d5432.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
One more note, it is very helpful for us if the release notes have descriptive names, such as `remove-mpl-pulse` (the important part is that it says "remove" and what it is removing). This is not a hard requirement, but as I said, helpful to later locate the relevant release notes.